scripts/launch-distributed-training.sh

```
@@ -0,0 +1,126 @@
+    echo ""
+    echo "Examples:"
+    echo "  $0 4 ./MyTrainingApp"
+    echo "  $0 8 ./MyTrainingApp --epochs 100"
+    exit 1
+fi
+
+# Parse arguments
+NUM_PROCESSES=$1
+PROGRAM=$2
+shift 2
+PROGRAM_ARGS="$@"
```

Shellcheck (0.11.0)

[warning] Assigning an array to a string! Assign as array, or use * instead of @ to concatenate.

(SC2124)